src/metabase/config.clj

```diff
@@ -52,7 +52,9 @@
    :max-session-age        "20160"                                        ; session length in minutes (14 days)
    :mb-colorize-logs       (str (not is-windows?))                        ; since PowerShell and cmd.exe don't support ANSI color escape codes or emoji,
    :mb-emoji-in-logs       (str (not is-windows?))                        ; disable them by default when running on Windows. Otherwise they're enabled
-   :mb-qp-cache-backend    "db"})
+   :mb-qp-cache-backend    "db"
+   :mb-max-results         "10000"
+   :mb-default-max-results-bare-rows "2000"})
```

Comment:
You shouldn't need to specify these here if you specify the `:default` in the `defsetting` form, `defsetting` doesn't look at this and nothing else is looking for these env vars